linodecli/configuration/__init__.py

```diff
@@ -336,6 +336,9 @@ def configure(
         images = [
             i["id"] for i in _do_get_request(self.base_url, "/images")["data"]
         ]
+        engines = [
+            e["id"] for e in _do_get_request(self.base_url, "/engines")["data"]
```

Comment:
```suggestion
            e["id"] for e in _do_get_request(self.base_url, "/databases/engines")["data"]
```
404 error occurs during `linode configure`, I would suggest double check the API endpoint.